src/org/labkey/test/components/glassLibrary/grids/HeatMap.java

```diff
@@ -0,0 +1,211 @@
+package org.labkey.test.components.glassLibrary.grids;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.openqa.selenium.By;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.labkey.test.WebDriverWrapper.WAIT_FOR_JAVASCRIPT;
+import static org.labkey.test.util.TestLogger.log;
+
+public class HeatMap extends WebDriverComponent<HeatMap.ElementCache>
```

Comment:
Document the location of the wrapped product component
```suggestion

/**
 * Manipulates the HeatMap component from labkey-ui-components (components/src/internal/components/heatmap/HeatMap.tsx)
 */
public class HeatMap extends WebDriverComponent<HeatMap.ElementCache>
```